### ISSUE TITLE
nn.ConvTranspose2d output_padding not padding 0

### ISSUE THREAD

**Issue by SongLongZhang** [pytorch/pytorch#25536]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)
 m = nn.ConvTranspose2d(1, 1, (2, 2), stride=(2, 2), padding=(1, 1), output_padding=(1,1))
 output = m(input)
print(output)
tensor([[[[ 0.3960,  0.6212,  0.3960,  0.6212,  0.3960,  0.6212,  0.3960],
          [-0.2565,  0.0108, -0.2565,  0.0108, -0.2565,  0.0108, -0.2565],
          [ 0.3960,  0.6212,  0.3960,  0.6212,  0.3960,  0.6212,  0.3960],
          [-0.2565,  0.0108, -0.2565,  0.0108, -0.2565,  0.0108, -0.2565],
          [ 0.3960,  0.6212,  0.3960,  0.6212,  0.3960,  0.6212,  0.3960],
          [-0.2565,  0.0108, -0.2565,  0.0108, -0.2565,  0.0108, -0.2565],
          [ 0.3960,  0.6212,  0.3960,  0.6212,  0.3960,  0.6212,  0.3960]]]],
why output_padding not padding 0 to out?What number is pad?

**Comment by SongLongZhang** [comment]
How can I replace ConvTranspose2d（output_padding） with ConvTranspose2d(no output_padding)?I want to ConvTranspose2d with output_padding in MXNet ?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions
